src/Infrastructure/Controller/UserController.php

```diff
@@ -6,23 +6,50 @@
 
 use App\Application\Service\UserService;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Attribute\Route;
 
+
 class UserController extends AbstractController
 {
     public function __construct(
         private readonly UserService $userService,
     ) {}
 
     #[Route('/user', name: 'create_user')]
```

Comment:
Nézz utána légyszi, hogy hogyan tudunk query paramétert validálni, amit lehet annotációban, itt van két segítség:
https://symfony.com/blog/new-in-symfony-6-3-query-parameters-mapper
https://symfony.com/blog/new-in-symfony-6-3-mapping-request-data-to-typed-objects

A fenti doksikban találsz példákat, arra figyelj, hogy az `attributes` füleket válaszd, ahol van lehetőség, és akkor a példa az annotációban lesz elírva.

Legyen:

- email, kötelező, email 
- name kötelező, string, minimum 10 karakter maximum 50
- age, kötelező, int, minimum 18 max 99